.vib/zookeeper/vib-verify.json

```diff
@@ -38,6 +38,17 @@
             "vuln_type": ["OS"]
           }
         },
+        {
+          "action_id": "goss",
+          "params": {
+            "resources": {
+              "path": "/.vib/zookeeper/goss"
+            },
+            "remote": {
+              "workload": "sts-zookeeper"
+            }
+          }
+        },
         {
           "action_id": "health-check",
```

Comment:
Make sure the order of the actions is the same in both `vib-verify` and `vib-publish` pipelines. Right now it´s different. 
1. trivy
2. goss
3. health check